source/AI.cpp

```diff
@@ -3489,7 +3549,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 		}
 	}
 	
-	if(ship.HasBays() && isLaunching)
+	if(ship.HasBays() && IsLaunching(ship))
```

Comment:
Perhaps to reduce some ambiguity, we name this method `HasDeployingFighter` or similar.

Reply:
Was indeed a bit ambiguous, renamed to `HasDeployments`.